lib/LMSManagers/LMSCustomerManager.php

```diff
@@ -824,6 +829,16 @@ public function GetCustomer($id, $short = false)
                     if ($types)
                         $result['contacts'][$idx]['typestr'] = implode('/', $types);
                 }
+            if (is_array($result['emails']))
+                foreach ($result['emails'] as $idx => $row) {
+                    $types = array();
+                    foreach ($CONTACTTYPES as $tidx => $tname)
+                        if ($row['type'] & $tidx)
+                            $types[] = $tname;
+
+                    if ($types)
+                        $result['emails'][$idx]['typestr'] = implode('/', $types);
+                }
```

Comment:
Co ta zmiana ma na celu? Poprawkę jakiegoś błędu? Jeśli tak to lepiej zupełnie odrębnym pull requestem to zrobić i opisać w tytule commit-a.